Code/Framework/AzCore/AzCore/Metrics/JsonTraceEventLogger.cpp

```diff
@@ -247,6 +247,7 @@ namespace AZ::Metrics
         {
             extraParams.emplace_back(ThreadDurationKey, EventValue{ AZStd::in_place_type<AZ::s64>, completeArgs.m_tdur->count() });
         }
+        eventDesc.SetExtraParams(extraParams);
```

Comment:
I'll fix this in my https://github.com/o3de/o3de/pull/12476/files PR as all the functions need to have a SetExtraParams call